example/Components/SimpleForm.js

```diff
@@ -28,27 +28,47 @@ const Form = ({
 (
   <div className='form'>
     <div className='formGroup'>
-      <label>Name</label>
+      <label htmlFor='name'>Name</label>
```

Comment:
All labels must be associated with their input either implicitly (wrapping the input) or explicitly (using the `for` attribute).